### PR TITLE
Enum moved form schemas/common.py to schemas/revisions

### PR DESCRIPTION
**Pull request by manicdon7**
Closes #

## Introduced changes

Enum moved form schemas/common.py to schemas/revisions
- 
- 

##

- [ ] This PR contains breaking changes

### CODE REVIEW

**Review by franciszekjob — CHANGES_REQUESTED** [review]
Please fix tests and lint.

File: pyproject.toml
Comment (franciszekjob) [inline pyproject.toml:1]: Please don't change `pyright` version.

File: requirements.txt
Comment (franciszekjob) [inline requirements.txt:1]: Not needed.

**Review by franciszekjob — CHANGES_REQUESTED** [review]

File: pyproject.toml
Comment (manicdon7) [reply pyproject.toml:1]: Sure I'll change that to previous version

File: requirements.txt
Comment (manicdon7) [reply requirements.txt:1]: Sure I'll remove it
and again raise a PR

**Comment by manicdon7** [comment]
updated the changes

**Comment by manicdon7** [comment]
I have passed the required tests and linting. Still my request has been
closed.

**Comment by franciszekjob** [comment]
You've modified dependency version which was not needed.

**Comment by manicdon7** [comment]
If I kept same version it was ended in test fail.

**Pull request closed without merging**